app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+  include TasksHelper
+
+  skip_before_action :set_project, unless: -> { current_project }
+  skip_before_action :set_nodes, unless: -> { current_project }
```

Comment:
These lines are at odds w/ L2, no? Under what conditions we won't have a project?